plugins/catalog-backend/src/next/NextRouter.ts

```diff
@@ -142,6 +142,11 @@ export async function createNextRouter(
         const output = await locationService.getLocation(id);
         res.status(200).json(output);
       })
+      .get('/locations/:type/refresh', async (req, res) => {
```

Comment:
So i'm not sure about the API and REST endpoint here.

I think this opens up a bigger discussion here to what we want to public API of this to be.

Should be it be a `location` or could it be a single `entityRef` and then it can refresh the parent location?

Reply:
If we're opening up that question, I would probably say "both", i.e. create both endpoints, unless we can come up with a good reason not to, just for the flexibility.